samples/DataSourceWithProcessingAndExtensions/Sample/MetadataTables/EventStatsTable.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Performance.SDK.Extensibility;
+using Microsoft.Performance.SDK.Processing;
+using DataExtensionsSample.SourceDataCookers;
+
+namespace DataExtensionsSample.MetadataTables
+{
+    [Table]
+    [RequiresCooker(EventStatsCooker.CookerPathAsString)]
+    public class EventStatsTable
```

Comment:
>public class EventStatsTable [](start = 4, length = 28)

What's the difference between a Metadata table and a regular table? Maybe a class docstring explaining that difference would be useful.